code/src/cljs/sixsq/nuvla/ui/i18n/dictionary.cljs

```diff
@@ -1035,6 +1055,7 @@
         :bulk-force-delete-in-progress                 "Batch de suppressions forcées en cours"
         :bulk-stop-in-progress                         "Batch d'arrêts en cours"
         :bulk-update-in-progress                       "Batch de mises à jour en cours"
+        :bulk-update-tags                              "Mise à jour des mots clés en batch"
```

Comment:
keep tags word. Clearer and team prefer this. Look where tags are used and make them consistent.